### ISSUE TITLE
:8080/my-account is only loading loop

### ISSUE THREAD

**Issue by Jackiiiii** [scottyzen/woonuxt#172]
Hello, i wanted to ask if u maybe know why my account page has an loading loop. i have install 3 wordpress, reinstall it on 2 server and still the same everything else works.

kind regards
Jack

**Comment by Jackiiiii** [comment]
https://github.com/scottyzen/woonuxt/assets/33198895/8bc0808c-6964-40e4-8e1f-6220066c47c0

**Comment by scottyzen** [comment]
Hi @Jackiiiii,

Are there any errors in the console?

**Comment by Jackiiiii** [comment]
If i start the server, in the console i see no error, or do you mean somewhere else?

**Comment by scottyzen** [comment]
Right-click on the page and select `inspect` and then go to the `console` tab.

**Comment by alexookah** [comment]
@scottyzen 
I managed to reproduce the problem! My account waits for cart to load

and defineNuxtPlugin in plugin.init.ts
has this code:
```
    // Wait for the user to interact with the page before refreshing the cart, this is helpful to prevent excessive requests to the server
    let initialised = false;
    const eventsToFireOn = ['mousedown', 'keydown', 'touchstart', 'scroll', 'wheel', 'click', 'resize', 'mousemove', 'mouseover'];
```
Video:
https://github.com/user-attachments/assets/7aef2f56-ba7f-4772-9234-23f71887aa16


Unless there is some user interaction the cart wont try to load.
The same problem occurs also in checkout page. If you try to refresh without any event triggering

I dont think it makes much sense with this events toFireOn. 
Maybe it should be configurable. But I would not enable this in my personal project.
I think this code should be removed. Because in mobile if someone tries to open a page they might be waiting forever until they try to scroll up and down for the request to trigger.

WDYT?
Should I make a PR removing this?

**Comment by alexookah** [comment]
@scottyzen @Jackiiiii 
this issue should be resolved now.
